.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+
+| Q               | A
+| --------------- | ---
+| Laravel version | 5.1 / 5.2
+| PHP version     | 5.6.x
```

Comment:
LOL, yeah crap !
_Laravel version: PHP (I think the thing my developer uses finishes with a P)_

@taylorotwell will fine-tune that